README.md

```diff
@@ -69,14 +69,15 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 ## Prerequisites
 
 * Ruby 2.3+
-* Your cluster must be running Kubernetes v1.8.0 or higher<sup>1</sup>
+* Your cluster must be running Kubernetes v1.9.0 or higher<sup>1</sup>
 * Each app must have a deploy directory containing its Kubernetes templates (see [Templates](#using-templates-and-variables))
 * You must remove the` kubectl.kubernetes.io/last-applied-configuration` annotation from any resources in the namespace that are not included in your deploy directory. This annotation is added automatically when you create resources with `kubectl apply`. `kubernetes-deploy` will prune any resources that have this annotation and are not in the deploy directory.<sup>2</sup>
 * Each app managed by `kubernetes-deploy` must have its own exclusive Kubernetes namespace.
 
-<sup>1</sup> We run integration tests against these Kubernetes versions. Kubernetes v1.7 was officially
-supported in gem versions < 0.21. Kubernetes v1.6 was officially supported in gem versions < 0.16. Kubernetes
-v1.5 was officially supported in gem versions < 0.12.
+<sup>1</sup> We run integration tests against these Kubernetes versions.
+Kubernetes v1.8 was officially supported in gem versions < 0.22. Kubernetes v1.7 was
+officially supported in gem versions < 0.21. Kubernetes v1.6 was officially supported in gem
+versions < 0.16. Kubernetes v1.5 was officially supported in gem versions < 0.12.
 
 <sup>2</sup> This requirement can be bypassed with the `--no-prune` option, but it is not recommended.
```

Comment:
Maybe you're not done updating the readme, but in case you missed them, there are additional 1.8 references still further down.

Reply:
No, I missed them. I assumed the grep hits were all coming from this paragraph.